### ISSUE TITLE
🐞 Bug Report: Elementor PRO 3.16.2 breaks WP-TOOLKIT again like in #21431

### ISSUE THREAD

**Issue by furchtlos76** [elementor/elementor#24063]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Elementor Elementor Pro 3.16.2

When using using Wordpress tookit in PLESK, it throws this error when trying to use "login to wordpress" like in Bug Report  #21431 but in WP Toolkit for PLESK and only affecting when Elementor PRO is active!

"Error getting administrator credentials: An error has occurred when decoding JSON: illegal Token"

If disabling elementor PRO and only let elementor active, problem goes away. Going back to php 8.1 or 7.4 fixes the problem too!

Steps to reproduce

Use PHP 8.2.1 for a Domain (some report they had the problem when updating to 8.1. - i directly gone to 8.2.1)
Try to login to Wordpress via WP-Toolkit in PLESK there are several reports in Plesk, too like:

https://support.plesk.com/hc/en-us/articles/13302778467991
https://talk.plesk.com/threads/error-getting-administrator-credentials-illegal-token-when-decoding-json.368755/

### Steps to reproduce

Use PHP 8.2.1 for a Domain (some report they had the problem when updating to 8.1. - i directly gone to 8.2.1)
Try to login to Wordpress via WP-Toolkit in PLESK there are several reports in Plesk, too like:

https://support.plesk.com/hc/en-us/articles/13302778467991
https://talk.plesk.com/threads/error-getting-administrator-credentials-illegal-token-when-decoding-json.368755/

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: mariadb.org binary distribution v10.11.5-MariaDB-1:10.11.5+maria~ubu2004
	PHP Version: 8.2.11
	PHP Memory Limit: 512M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 16M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.3.2
	Site URL: https://www.xxx
	Home URL: https://www.xxxx
	WP Multisite: No
	Max Upload Size: 16 MB
	Memory limit: 40M
	Max Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: de-DE
	Timezone: 1
	Admin Email: info@xxxx
	Debug Mode: Inactive
	
		Elementor
		Version: 3.16.6
		Author: Elementor.com

	Elementor Pro
		Version: 3.16.2
		Author: Elementor.com

== Elementor-Experimente ==
	Optimierte DOM Ausgabe: Inaktiv
	Verbessertes Laden von Assets: Inaktiv
	Verbessertes Laden von CSS: Aktiv
	Inline-Schriftarten-Symbole: Inaktiv
	Zusätzliche benutzerdefinierte Breakpoints: Inaktiv
	admin_menu_rearrangement: Standardmäßig deaktiviert
	Flexbox Container: Inaktiv
	Upgrade Swiper Library: Inaktiv
	Grid Container: Inaktiv
	Default to New Theme Builder: Inaktiv
	Editor Top Bar: Inaktiv
	Startseiten: Inaktiv
	Nested Elements: Inaktiv
	Hintergrundbilder nachladen: Inaktiv
	Global Style Guide: Standardmäßig deaktiviert
	Page Transitions: Inaktiv
	Notes: Inaktiv
	Form Submissions: Inaktiv
	Scroll Snap: Inaktiv
	Menü: Standardmäßig deaktiviert
	Taxonomy Filter: Inaktiv
	
	== Protokoll ==
	
Log: showing 20 of 432023-09-20 16:00:34 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.16.3',
  'to' => '3.16.4',
)]
2023-09-20 20:38:20 [info] elementor-pro::elementor_pro_updater Started 
2023-09-20 20:38:20 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-09-20 20:38:20 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-09-20 20:38:20 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.16.1',
  'to' => '3.16.2',
)]
2023-09-20 20:38:20 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.16.1',
  'to' => '3.16.2',
)]
2023-10-09 16:00:44 [info] elementor::elementor_updater Started 
2023-10-09 16:00:44 [info] Elementor/Upgrades - _on_each_version Start  
2023-10-09 16:00:45 [info] Elementor/Upgrades - _on_each_version Finished 
2023-10-09 16:00:45 [info] Elementor/Upgrades - _v_3_17_0_site_settings_updates Start  
2023-10-09 16:00:45 [info] Elementor/Upgrades - _v_3_17_0_site_settings_updates Finished 
2023-10-09 16:00:45 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.16.4',
  'to' => '3.16.5',
)]
2023-10-09 16:00:45 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.16.4',
  'to' => '3.16.5',
)]
2023-10-18 04:00:43 [info] elementor::elementor_updater Started 
2023-10-18 04:00:43 [info] Elementor/Upgrades - _on_each_version Start  
2023-10-18 04:00:43 [info] Elementor/Upgrades - _on_each_version Finished 
2023-10-18 04:00:43 [info] Elementor/Upgrades - _v_3_17_0_site_settings_updates Start  
2023-10-18 04:00:43 [info] Elementor/Upgrades - _v_3_17_0_site_settings_updates Finished 
2023-10-18 04:00:43 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.16.5',
  'to' => '3.16.6',
)]
2023-10-18 04:00:43 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.16.5',
  'to' => '3.16.6',
)]

== Elementor - Compatibility Tag ==
	
	Elementor Pro: Die Kompatibilität ist nicht angegeben
	Essential Addons for Elementor: Die Kompatibilität ist nicht angegeben
	Essential Addons for Elementor - Pro: Die Kompatibilität ist nicht angegeben

== Elementor Pro - Compatibility Tag ==

**Comment by UltimateByte** [comment]
Plesk's WP Toolkit team is supposed to be in touch with Elementor team as said in the Plesk group on Facebook. That is probably the way to go for this...

**Comment by milosz-rycko-bozenski** [comment]
Yeah, I ran into this issue on few sites I created and I host. I thought that it was Plesk's WP Toolkit bug.

Sorry, but nope.

It was issue with my code. After fixing my code, everything is ok.

Double check your theme files and custom codes.

**Comment by Paradinight** [comment]
Quick Fix

File: elementor-pro/core/editor/editor.php
Go To Line 118 and Change ${package} to {$package}.

${package} is deprecated.

**Comment by furchtlos76** [comment]
> Yeah, I ran into this issue on few sites I created and I host. I thought that it was Plesk's WP Toolkit bug.
> 
> Sorry, but nope.
> 
> It was issue with my code. After fixing my code, everything is ok.
> 
> Double check your theme files and custom codes.

maybe you had another bug... fix from @Paradinight worked great!

**Comment by furchtlos76** [comment]
> Quick Fix
> 
> File: elementor-pro/core/editor/editor.php Go To Line 118 and Change ${package} to {$package}.
> 
> ${package} is deprecated.

Thanks alot! That fix did the trick, even Elementor Support still tells me, its a WP Toolkit bug and they can not help.... really disapointing

**Comment by UltimateByte** [comment]
So in the end there is an actual error in Elementor, but I still wonder... How can it affect WP Toolkit unless it has structural deficiencies that can let it be affected by WordPress's plugins?

**Comment by Paradinight** [comment]
@UltimateByte WP Toolkit has the display_errors active and shows all errors, warnings and notice.

https://www.php.net/manual/en/errorfunc.configuration.php#ini.display-errors

**Comment by UltimateByte** [comment]
> @UltimateByte WP Toolkit has the display_errors active and shows all errors, warnings and notice.
> 
> https://www.php.net/manual/en/errorfunc.configuration.php#ini.display-errors 

Yes OK but it prevents login to WP admin in some contexts... So it's not only a matter of displaying errors but also its behavior is affected somehow.

**Comment by glennonline** [comment]
When wil this get fixed? as I think it's now proven that the bug is with elementor.

**Comment by furchtlos76** [comment]
> When wil this get fixed? as I think it's now proven that the bug is with elementor.

I am interested to, but as i gut other errors and elementor support always tells me its the problem of others and they can not support other plugins, i guess it wioll take ages... sorry but that support should shame for naming it support :-/

**Comment by tony2476** [comment]
I talked with Elementor Techinical Support today and **they pointed the finger at WP Toolkit** as the problem again. Very frustrating that they will not fix their software and update their **deprecated code**.

**Comment by furchtlos76** [comment]
> I talked with Elementor Techinical Support today and **they pointed the finger at WP Toolkit** as the problem again. Very frustrating that they will not fix their software and update their **deprecated code**.

Haha, as i said, their support has no plan! And the developers seem not to care, even it is only 1 line to fix and the fix is already clear...

**Comment by UltimateByte** [comment]
@furchtlos76 Maybe try to make the pull request with the fix? 
Add the tag `Fixes #24063` to the pull request :)

**Comment by UltimateByte** [comment]
@furchtlos76 I was up to do it, but in fact, my bad, Elementor Pro is not publicly on Git so we can't even do that.

**Comment by furchtlos76** [comment]
God thanks Plesk fixed the bug elementor should fix, as here none seems to care, even of such easy to do fixes. Whats up?!

**Comment by Paradinight** [comment]
@furchtlos76 Elementor Pro 3.18 Beta has the fix.
